cpp/src/io/parquet/decode_fixed.cu

```diff
@@ -1285,6 +1252,16 @@ CUDF_KERNEL void __launch_bounds__(decode_block_size_t, 8)
   if constexpr (has_strings_t) {
     // For large strings, update the initial string buffer offset to be used during large string
     // column construction. Otherwise, convert string sizes to final offsets.
+
+    if constexpr (!has_lists_t) {
+      if (!should_process_nulls) {
+        if (t == 0) {
+          s->nesting_info[s->col.max_nesting_depth - 1].value_count = s->input_row_count;
+        }
+        block.sync();
+      }
+    }
```

Comment:
Maybe I overlooked the lines that used to set this, but why do we need to set it here now?